packages/block-editor/src/components/offline-status/index.native.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { Text, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+import { Icon } from '@wordpress/components';
+import { offline as offlineIcon } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style';
+
+const OfflineStatus = () => {
+	const containerStyle = usePreferredColorSchemeStyle(
+		styles.container,
+		styles[ 'container--dark' ]
+	);
+
+	const textStyle = usePreferredColorSchemeStyle(
+		styles.text,
+		styles[ 'text--dark' ]
+	);
+
+	const iconStyle = usePreferredColorSchemeStyle(
+		styles.icon,
```

Comment:
Noting that the linter will throw an error if these are written in square-bracket syntax, e.g., `styles[ 'icon' ]`:

```
  21:11  error  ["container"] is better written in dot notation  dot-notation
  26:11  error  ["text"] is better written in dot notation       dot-notation
  31:11  error  ["icon"] is better written in dot notation       dot-notation
```

Perhaps a quirk of BEM semantics, but using dot-notation for light styles and square-bracket syntax for dark styles, unless you have any suggestions.